app.js

```diff
@@ -0,0 +1,64 @@
+const express = require('express')
+const cors = require('cors')
+
+ const port = 3000;
+
+ 
+const app = express()
+ app.use(cors())
+ 
+ const animalArray = ['lion','zebra','monkey','dinosaur','fish','cat','lizard']
+
+ let randomNum
```

Comment:
You should have `let randomNum` a `;` to know that thats what you meant to do, since right here it just looks like you forgot to finish that part of the code.